test/integration/test_collections.py

```diff
@@ -320,6 +324,7 @@ def test_get_datasets(conservator):
 def test_download_datasets(conservator, tmp_cwd):
     collection = conservator.collections.create_from_remote_path("/Some/Collection")
     dataset_1 = collection.create_dataset("My first dataset")
+    assert wait_for_dataset_commit(conservator, dataset_1.id)
     # Wait for the server to create and commit the dataset.
```

Comment:
Didn't you mean to replace the "Wait for the server to create and commit the dataset" section with wait_for_dataset_commit()?

Reply:
Removed redundant code.